### DISCUSSION TITLE
Consume a11ysupport package

### DISCUSSION THREAD

**Discussion by Ryuno-Ki** [mdn/mdn-community#284] [Platform]
Over in  https://github.com/mdn/content/issues/7445 we pondered the possibility to consume https://www.npmjs.com/package/a11ysupport to help us maintain ARIA / AT pages on MDN.

I've contacted the author in https://github.com/accessibilitysupported/a11ysupport.io/issues/179 and he's open to the idea.

I haven't looked into what's needed exactly, but want to fill an issue, so I don't forget about it here.

Also, be aware, that the package itself might get be superseded by a W3C Community Group effort. Eric is keeping us in the loop if he learns more about it.

**Comment by Malvoz** [comment]
Excuse my ignorance, I thought all support data on MDN came from https://github.com/mdn/browser-compat-data/. There's a related issue: https://github.com/mdn/browser-compat-data/issues/6553, if that's relevant somehow. 😋
